packages/jetpack-blocks/babel.config.js

```diff
@@ -0,0 +1,89 @@
+/** @format */
+const _ = require( 'lodash' );
+
+const targets = { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] };
+
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				targets,
+				useBuiltIns: 'entry',
+				// allows es7 features like Promise.prototype.finally
+				shippedProposals: true,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
+			},
+		],
+		'@babel/react',
+	],
+	plugins: _.compact( [
```

Comment:
We can drop the `_.compact()` -- not falsey array items to filter here :slightly_smiling_face:

Reply:
https://github.com/Automattic/wp-calypso/pull/31503/commits/d544f2b3f5ff756688995033d544fd6d7f549e1b